src/components/ItaliaTheme/Blocks/NumbersBlock/Edit.jsx

```diff
@@ -48,7 +48,33 @@ class Edit extends SubblocksEdit {
   constructor(props) {
     super(props);
     this.state.selectedField = 'title';
+
+    this.state = {
+      ...this.state,
+      icon1: '',
+      icon2: '',
+      icon3: '',
+    };
```

Comment:
```suggestion
    this.state = {
      selectedField: 'title',
      icon1: '',
      icon2: '',
      icon3: '',
    };
```
Già che ci siamo mettiamo tutto assieme, diventa più leggibile

Reply:
Così si rompe, manca una parte dello stato (credo subblock qualcosa?)

Reply:
Ah perché estende un altro componente, cosa che di solito non si fa, non l'avevo notato. Ok allora nevermind